plutus-core/generators/Language/PlutusCore/Generators/Interesting.hs

```diff
@@ -142,19 +143,19 @@ genNaiveFib = do
 -- defined in terms of generic fix (see 'Fix') and return the result
 -- along with the original 'Integer'
 genNatRoundtrip
-    :: forall uni. Generatable uni => TermGen uni Integer
+    :: TermGen Integer
 genNatRoundtrip = do
-    let typedInt = AsKnownType @(Term TyName Name uni ())
+    let typedInt = AsKnownType @(Term TyName Name DefaultUni DefaultFun ())
     TermOf _ iv <- Gen.filter ((>= 0) . _termOfValue) $ genTypedBuiltinDef typedInt
     let term = mkIterApp () natToInteger [metaIntegerToNat iv]
     return $ TermOf term iv
 
 -- | @sumNat@ as a PLC term.
-natSum :: uni `Includes` Integer => Term TyName Name uni ()
+natSum :: Term TyName Name DefaultUni DefaultFun ()
```

Comment:
Lots of these type sigs seems to have become simpler - where they overly general before or did something get simpler?

Reply:
They were more general. Previously `natSum` worked over any `uni` that has `Integer` in it, now it only works in `DefaultUni`. There isn't much point in being general over the universe if you're not general over the set of built-in functions and I did not want to be general over the latter as that would be too much trouble for nothing at the moment.